.github/workflows/test.yaml

```diff
@@ -73,6 +73,11 @@ jobs:
           with:
               python-version: ${{ matrix.python-version }}
 
+        # TODO: Remove once dependency work is completed
+        # https://github.com/ApeWorX/ape/issues/1327
+        - name: Copy smart-contracts to dependencies
```

Comment:
I think if you do an `if: ` clause here, you can just do it once, so it doesn't fail